src/requirements/types.ts

```diff
@@ -28,6 +28,7 @@ type GenericRequirementsJson<R> = {
   readonly college: CollegeRequirements<R>;
   readonly major: MajorRequirements<R>;
   readonly minor: MajorRequirements<R>;
+  readonly grad: MajorRequirements<R>;
```

Comment:
Maybe add a comment here that says we're treating grad programs at the same level as Major Requirements?